resources/qml/WelcomePages/CloudContent.qml

```diff
@@ -135,16 +176,28 @@ Item
                 }
             }
 
-            Cura.SecondaryButton
+            // Create an account link
+            Label
             {
-                id: createAccountButton
+                id: createAccountLabel
+                width: parent.width
                 anchors.horizontalCenter: parent.horizontalCenter
-                text: catalog.i18nc("@button","Create account")
-                onClicked: Qt.openUrlExternally(CuraApplication.ultimakerCloudAccountRootUrl + "/app/create")
+                horizontalAlignment: Text.AlignHCenter
+                text:
+                {
+                    var t1 = catalog.i18nc("@text", "Create a free Ultimaker Account")
+                    var t = "<a href='https://notusedref'>" + t1 + "</a>"
+                    return t
+                }
```

Comment:
Since the whole text is the same link, what's wrong with using the `onClicked` event like before? It wouldn't need to use all that rich text formatting then.

We normally use the `TertiaryButton` QML element for this sort of thing. It is styled to look like a link. Re-using that would make it look consistent with the rest of Cura's interface regardless of theme.